se/se_epub.py

```diff
@@ -508,6 +508,28 @@ def recompose(self, output_xhtml5: bool, extra_css_file: Path = None) -> str:
 
 			file_css = regex.sub(r"\s*@(charset|namespace).+?;\s*", "\n", file_css).strip()
 
+			# Convert background-image URLs (from local.css only) to base64
+			if filename == "local.css":
+				convert_css = file_css
+				for img in regex.finditer(pattern=r'(url\(".*\.)(svg)("\))', string=convert_css):
```

Comment:
It looks like this regex is only going to catch SVG files, so I'm not sure why we test if the file is SVG/JPG/PNG later on. We can probably improve this regex by saying `r"url\(\"(.+?\.(?:svg|png|jpg))\"\)"` which should capture the entire filename including extension in the first capture group. That way we can correctly test the file extension, and we won't have to do further substitution on line 516 to be able to use it directly.

Reply:
That's a very good point. I probably meant to catch all three. I will change and submit after I respond to the second item as well